types/office-js/index.d.ts

```diff
@@ -2102,6 +2102,15 @@ declare namespace Office {
          * Returns string values that match the named regular expression defined in the manifest XML file
          */
         getRegExMatchesByName(name: string): Array<string>;
+        /**
+        * Gets the entities found in the selected item that are currently selected (1.6 and up)
+        */
+        getSelectedEntities(): Entities;
+        /**
+         * Returns string values in the currently selected message object that match the regular expressions defined in the manifest XML file and
+         * are selected in the current item (1.6 and up)
```

Comment:
Both for this comment and the other one, could you expand the "1.6" text to clarify the requirement set that it's associated with?  E.g., `Minimum requirement set: Mailbox 1.6`